command/operator_api.go

```diff
@@ -198,29 +202,24 @@ func (c *OperatorAPICommand) Run(args []string) int {
 		c.Ui.Output(out)
 		return 0
 	}
-
-	// Re-implement a big chunk of api/api.go since we don't export it.
-	client := cleanhttp.DefaultClient()
-	transport := client.Transport.(*http.Transport)
-	transport.TLSHandshakeTimeout = 10 * time.Second
-	transport.TLSClientConfig = &tls.Config{
-		MinVersion: tls.VersionTLS12,
-	}
-
-	if err := api.ConfigureTLS(client, config.TLSConfig); err != nil {
-		c.Ui.Error(fmt.Sprintf("Error configuring TLS: %v", err))
-		return 1
-	}
+	apiR := apiC.Raw()
 
 	setQueryParams(config, path)
-
-	verbose("> %s %s", c.method, path)
+	verboseSocket(config, fmt.Sprintf("*   Trying %s:0...", config.SocketPath))
 
 	req, err := http.NewRequest(c.method, path.String(), c.body)
 	if err != nil {
 		c.Ui.Error(fmt.Sprintf("Error making request: %v", err))
 		return 1
 	}
+	var h, p string
+	h = req.URL.Hostname()
+	if p = req.URL.Port(); p == "" {
+		p = "80"
+	}
+
+	verboseSocket(config, fmt.Sprintf("* Connected to %s (%s) port %s (#0)", h, config.SocketPath, p))
+	verbose("> %s %s %s", c.method, req.URL.Path, req.Proto)
```

Comment:
Like above I'm not sure what the port handling code is doing here. It looks like `p` is only used if a socket is used, and I don't think it needs to be printed at all in that use case. I'd say just drop it if it's never going to print anything useful.